### ISSUE TITLE
Submission button not working on page 3 with Firefox.

### ISSUE THREAD

**Issue by devrodmastria** [SpartaHack/SpartaHackVI-Website#11]
I attempted to submit my application three times without success using Firefox 73.0.1 (64-bit) on macOS Catalina. The fourth time worked as expected. This is regarding the page with id "application.html#p3/3"

**Comment by jarek-t** [comment]
Hi, thanks for letting me know about this! Those clicks were with/without reload? If without, do you remember the approximate duration of your click interval?

**Comment by devrodmastria** [comment]
The clicks were without reload. I'd say the interval was one second. I clicked the first time, nothing happened (visually). Clicked another time and looked for any sign of Firefox blocking any pop-ups... nothing again. So I thought I had exceeded the length limit for my personal statement. I took a few seconds to edit my response and noticed the hint "> 25 characters", so I thought I was OK with more than 25 characters. Thought about using Google Chrome, but finally decided to try the done button again after a few seconds and finally got through and saw the pop-up asking for consent to the MLH terms.
